src/scripts/app-controller.js

```diff
@@ -183,7 +183,10 @@ export default class AppController {
     console.log('Sending XHR to Web Push Protocol endpoint');
 
     fetch(subscription.endpoint, {
-      method: 'post'
+      method: 'post',
+      headers: {
+        'TTL': '60',
```

Comment:
Linter is throwing up on the TTL being quoted, could you unquote it (Sorry for asking for such a minor thing)